paasta_tools/cli/cmds/local_run.py

```diff
@@ -653,6 +678,89 @@ def check_if_port_free(port):
     return True
 
 
+def assume_aws_role(
+    instance_config, service, assume_role, assume_pod_identity, aws_profile
+):
+    """Runs AWS cli to assume into the correct role, then extract and return the ENV variables from that session"""
+    pod_identity = instance_config.get_iam_role()
+    if assume_role:
+        pod_identity = assume_role
+    if assume_pod_identity and not pod_identity:
+        print(
+            "Error: --assume-pod-identity passed but no pod identity was found for this instance",
+            file=sys.stderr,
+        )
+        sys.exit(1)
+    default_profile = ""
+    if not aws_profile:
+        with open("/nail/etc/ecosystem") as ecosystem_file:
+            default_profile = ecosystem_file.read()
+    profile = aws_profile or default_profile
+    if pod_identity and assume_pod_identity:
+        print(
+            "Attempting to assume role {} using profile {}".format(
+                pod_identity, profile
+            )
+        )
+        if os.getuid() == 0:
+            assume_role_cmd = "sudo -u {} HOME=/nail/home/{} aws --profile {} sts assume-role --role-arn {} --role-session-name {}-{}"
+            assume_role_cmd = assume_role_cmd.format(
+                get_username(),
+                get_username(),
+                profile,
+                pod_identity,
+                service,
+                get_username(),
+            )
+        else:
+            assume_role_cmd = "aws --profile {} sts assume-role --role-arn {} --role-session-name {}-{}"
+            assume_role_cmd = assume_role_cmd.format(
+                profile, pod_identity, service, get_username()
+            )
+        cmd = subprocess.run(assume_role_cmd, shell=True, stdout=subprocess.PIPE)
```

Comment:
I might be reading too quickly, but don't see how we ever grab the creds from this and return it? Should we be doing 
```
cmd_output = json.loads(cmd.stdout)
```
like we do below for cred_process_cmd when we pull creds w/o assuming a role?